gobblin-hive-registration/src/main/java/gobblin/hive/HiveNopSchemaManager.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2014-2016 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package gobblin.hive;
+
+import org.apache.hadoop.fs.Path;
+import org.apache.hadoop.hive.metastore.api.SerDeInfo;
+
+import gobblin.annotation.Alpha;
+import gobblin.configuration.State;
+
+
+/**
+ * A {@link HiveSchemaManager} that does nothing. This should be used when registering a table
+ * that does not require specifying schema properties.
+ *
+ * @author ziliu
+ */
+@Alpha
```

Comment:
I think the `Alpha` annotations are typically meant just for public APIs (e.g. interfaces or abstract classes), although there is no harm in tagging concrete class too.